Workaround for Cyclic Incompatible Dependencies on 3.0.x

gsteel opened the pull request:
Adds a script to check out the current tree as a 2.99.x branch for CI runs so that composer will install cyclic incompatible dependencies.

Ocramius commented on composer.lock:
```diff
@@ -4,7 +4,7 @@
         "This file is @generated automatically"
     ],
-    "content-hash": "a003c15d0428178d28dd954a18766ec9",
+    "content-hash": "3052afdac31561cdc43064a657f79b64",
```
This will be an issue with merge-up operations: unsure if I should first merge up `2.20.0`, or merge this and then fix it at merge time.

gsteel replied:
Ah yes, if we can get 2.20.x merged up, I can rebase and squash to get rid of all the noise

gsteel replied:
…actually, I've squashed and removed the commit that changes the lock file. No checks will run because no PHP code has changed but once 2.20 is in, everything should be groovy. If it's easier, I can just merge 2.20 into this pull

gsteel committed:
Adds a script to check out the current tree as a 2.99.x branch for CI runs so that composer will install cyclic incompatible dependencies.

Signed-off-by: George Steel <george@net-glue.co.uk>

Ocramius commented:
@gsteel the CI generated matrix looks really weird - is it bugged out?

gsteel commented:
@Ocramius - There's just zero checks because no *.php or *.md files have changed in this pull right? The checks ran before, but I squashed/forced pushed to clean up the pull - this is the last successful run before I squashed: https://github.com/laminas/laminas-view/actions/runs/1688557410

Ocramius commented:
Is it that? 🤔 

Really weird that it skips like that, but OK, I guess?

Ocramius commented:
Thanks @gsteel!
